app/src/main/res/layout/category_row_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/scoring_category_card_view"
+    android:layout_width="@dimen/category_card_width"
+    android:layout_height="@dimen/category_card_height"
+    app:cardElevation="@dimen/cardview_default_elevation"
+    app:cardCornerRadius="@dimen/cardview_default_radius"
+    >
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        >
+
+        <TextView
+            android:id="@+id/category_item_title"
+            style="@style/AvenirNext_DemiBold"
```

Comment:
You could use `android:fontFamily` instead, in case you want to add a different `style` on top of it. But maybe that's not YAGNI (not sure).